Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+import moment from 'moment';
+
+const HeaderTitle = () =>{
+    return(
+        <div className = "header-title">
+            <h1>Lambda School</h1>
+            <h2>@Lambda School &#8729; {moment().format('MMM D')}</h2>
```

Comment:
To match the exact format in the design file, you can use `.format('D MMM').toLowerCase()` instead.